src/format/formatDiagnosticMessage.ts

```diff
@@ -20,7 +20,7 @@ export const formatDiagnosticMessage = (message: string, format: (type: string)
   message
     // format declare module snippet
     .replaceAll(
-      /'(declare module )'(.*)';'/g,
+      /['“](declare module )['”](.*)['“];['”]/g,
```

Comment:
It should be better to replace all the `“` characters by `'` at the beginning in one replace instead of adding it to all the others.
WDYT?

Reply:
I thought the same way, but I can't guarantee that all  `“”` inside error messages use just as a replacement for `''`

Reply:
I didn't see `”` inside a type before, but I didn't use the Chinese language pack so we'll go with the safe option for now 🙂
Thank you!